raven_aiohttp.py

```diff
@@ -13,28 +13,36 @@
 import aiohttp
 import asyncio
 import socket
+import logging
 
 try:
     from asyncio import ensure_future
 except ImportError:
-    ensure_future = asyncio.async
-
+    ensure_future = getattr(asyncio, 'async')
```

Comment:
Why `asyncio.async` is not satisfactory?

Reply:
https://www.python.org/dev/peps/pep-0492/#deprecation-plans

Reply:
ok